.github/workflows/crucible-ci.yaml

```diff
@@ -69,23 +69,60 @@ jobs:
     - name: Echo generate-job-matrix-parameters outputs
       run: echo "${{ toJSON(needs.generate-job-matrix-parameters.outputs) }}"
 
-  crucible-ci-github-runners:
+  crucible-ci-build-controller:
     runs-on: ubuntu-latest
     needs:
     - generate-job-matrix-parameters
     - display-job-matrix-parameters
+    steps:
+    - name: checkout crucible-ci
+      uses: actions/checkout@v3
+      with:
+        repository: perftool-incubator/crucible-ci
+        ref: main
+        path: crucible-ci
+    - name: import secret
+      env:
+        CLIENT_SERVER_REGISTRY_AUTH_SECRET: ${{ secrets.CRUCIBLE_CI_ENGINES_REGISTRY_AUTH }}
+      if: ${{ needs.generate-job-matrix-parameters.outputs.build_controller == 'yes' && env.CLIENT_SERVER_REGISTRY_AUTH_SECRET != '' }}
+      run: sudo bash -c "echo \"$CLIENT_SERVER_REGISTRY_AUTH_SECRET\" > /root/crucible-ci-engines-token.json"
+    - name: run crucible-ci->install-crucible
+      if: ${{ needs.generate-job-matrix-parameters.outputs.build_controller == 'yes' }}
+      uses: ./crucible-ci/.github/actions/install-crucible
+    - name: display crucible config
+      if: ${{ needs.generate-job-matrix-parameters.outputs.build_controller == 'yes' }}
+      run: cat /etc/sysconfig/crucible
+    - name: verify crucible functionality
+      if: ${{ needs.generate-job-matrix-parameters.outputs.build_controller == 'yes' }}
+      run: sudo crucible repo info
+    - name: run crucible-ci->build-controller
+      if: ${{ needs.generate-job-matrix-parameters.outputs.build_controller == 'yes' }}
+      uses: ./crucible-ci/.github/actions/build-controller
+      with:
+        push_tag: crucible-${{ github.run_number }}
+    - name: no build required
+      if: ${{ needs.generate-job-matrix-parameters.outputs.build_controller == 'no' }}
+      run: echo "no controller build required"
```

Comment:
why do you have a "no build" case for the build job ?

Reply:
its really just a verbose logging step